bunsen/bunsen-avro/src/test/java/com/cerner/bunsen/avro/R4AvroConverterUsCoreTest.java

```diff
@@ -559,239 +569,6 @@ public void testCompile() throws IOException {
     }
   }
 
-  // TODO add test profile for R4: https://github.com/google/fhir-data-pipes/issues/558
```

Comment:
These tests have been uncommented and moved to [R4AvroConverterCustomProfileTest.java](https://github.com/google/fhir-data-pipes/pull/924/commits/edbba19c45f88f5d273720432c3a0b0a92e4e36f#diff-b988098f40473a9c057d544accfe28686ae807b135d8167dea47c2d96851af14)

Reply:
Yes, this should fix the issue #558. However, the test cases for both of them are identical now. We should also write some unit tests which are specific to each version, example testing the `R4StructureDefinitions.getReferenceTargetProfiles()` method